Core/Containers/DynamicsSelector.hpp

```diff
@@ -129,21 +135,28 @@ class DynamicsSelector : public DynamicsBase<
         return std::get<I>(models_);
     }
 
-private:
+    // private:
     // Helper function for retrieving the currently active model at runtime, by indexing the tuple
     // using template recursion. Since all models inherit from the same base type under this class,
     // their pointers can be casted to the base type
     template <size_t I = 0>
-    Base* GetActiveModel(){
-        if(I == index_){
+    Base *GetActiveModel()
+    {
+        if (I == index_)
+        {
             return &std::get<I>(models_);
-        } else if constexpr(I + 1 < std::tuple_size_v<std::tuple<Models...>>){
+        }
+        else if constexpr (I + 1 < std::tuple_size_v<std::tuple<Models...>>)
+        {
             return GetActiveModel<I + 1>();
-        } else{
+        }
+        else
+        {
```

Comment:
The auto formatter looks like it changed these lines. Please follow the existing formatting

Reply:
The format has been switch to the existing formatting